docs/devGuide/projectManagement.md

```diff
@@ -71,81 +71,47 @@ For general best practices, refer to the guide [_Working with PRs_ @SE-EDU](http
-   e.g., `git tag v2.0.1-markbind.XYZ`
-
-   <box type="important" seamless>
-
-   The tag used is a lightweight tag. _Don't_ use an annotated tag.
-   </box>
-
-1. **Push** everything to the main vue-strap repository (replace `XYZ` with version number).
-
-   ```
-   $ git push upstream master
-   $ git push upstream v2.0.1-markbind.XYZ
-   ```
-
 ### Building MarkBind
 
-1. **Navigate to the root** directory for MarkBind
-1. **Update dependencies** by running `npm install`
-1. **If we did a new release for the Vue components**:
-
-   Copy `vue-strap.min.js` from the vue-strap repository to the main asset folder, and to each of the test site's `expected/` folder.
-
-   ```sh
-   # copy to main asset folder
-   $ cp frontend/components/dist/vue-strap.min.js asset/js/vue-strap.min.js
-   ```
-   
-   Run `npm updatetest` and ensure only the minified bundle is copied to each of the expected test site's folders. If there are other changes, discard them for now - refer to **Rebuild the test files**.
-
-   Commit only the new vue-strap assets (replace `XYZ` with version number).
-
-   ```sh
-   $ git commit -m 'Update vue-strap version to v2.0.1-markbind.XYZ'
-   ```
-
-   <box type="info" seamless>
-
-   We uniquely do this for each MarkBind release (rather than spontaneously update the vue-strap files for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
-   </box>
+1. **Make sure to start with a "clean slate"** by running `npm run ci:all` or `npm run ciwin:all` if you haven't done so previously.
 
 1. **Increment the version number** by running `npm version`. Which to increment (`patch`, `minor` or `major`) depends on what PRs are merged for the new version, which means you must know beforehand about the changes.
 
    * If there are no significant changes, a `patch` is sufficient: `npm version patch`
    * If there are significant changes (e.g. breaking changes, new release), a `minor` release is needed:
      1. Run `npm version minor` as per normal.
-     1. Update the version number inside `src/lib/markbind/package-lock.json` and `src/lib/markbind/package.json` manually. This is because `npm version` will not automatically update the numbers from the outside. (Note: You do not have to commit these changes immediately, as we also have to rebuild the test files in the next step anyway.)
+        
+     1. Update the version number inside the `frontend/components` and `src/lib/markbind` packages similarly, as `npm version` does not update the subpackage versions.
 
    * We rarely do `major` releases, but if necessary, the steps are the same as the `minor` release (just change `minor` to `major`).
 
+   <box type="info" seamless>
+  
+   `npm version` creates a [version commit](https://docs.npmjs.com/cli/version) with an appropriate tag. We will make use of the generated tag later.
```

Comment:
1. Sounds good if we can do it all in one command, I'll try adding it in a separate commit for now, then squash to the npm scripts commit later. The drawback is needing to write the `--` in `npm version -- patch/major/minor`, but that's a really small thing anyway
2. 👍

Reply:
1. unfortunately it dosen't seem to be possible (not without a shell script or something like lerna https://stackoverflow.com/questions/51388921/pass-command-line-args-to-npm-scripts-in-package-json/51401577#answer-51401577), as passing arguments to npm scripts only passes it to the last command in the `... && ... && ...` chain. It's a small inconvenience though, I think we can do with it for now

Reply:
I did mean a shell script.

What is it about passing arguments?

Reply:
> I did mean a shell script.

ahh, in that case I'm not sure if we should do this, since we're going to need a `.bat` for windows as well.
If we're going there, we may as well consider a built in solution like lerna. 
Perhaps after restructuring `markbind/markbind-cli` to take full advantage of the structure and introducing lerna (since we're going to need to run tests on 3 packages it would be a good time - the amount of cd scripts would be rather big 👀 ) we could revisit this?
Releases aren't made often and I think dealing with the copy-paste + 2 commits process is sufficient to last us till then

Reply:
> What is it about passing arguments?

e.g. say we have ww: `npm run xx && npm run yy && npm run zz`
running `npm run ww -- minor` passes `minor` to `npm run zz` only 😞

Reply:
> ahh, in that case I'm not sure if we should do this, since we're going to need a `.bat` for windows as well.

On local machine, you can easily install and use Bash.

> Releases aren't made often and I think dealing with the copy-paste + 2 commits process is sufficient to last us till then

Ok.

> running npm run ww -- minor passes minor to npm run zz only 

Not sure why you would do that.

Reply:
> On local machine, you can easily install and use Bash.

that's one option 😮

> Not sure why you would do that.

thought you were meaning to do this initially, not the shell script ^